src/hotspot/share/utilities/vmError.cpp

```diff
@@ -1858,19 +1811,11 @@ void VMError::controlled_crash(int how) {
         fatal("Force crash with a nested ThreadsListHandle.");
       }
     }
-    case 18: {
-      // Check for assert when allocating from resource area without a
-      // ResourceMark.  There must not be a ResourceMark on the
-      // current stack when invoking this test case.
-      ResourceArea* area = Thread::current()->resource_area();
-      assert(area->nesting() == 0, "unexpected ResourceMark");
-      area->allocate_bytes(100);
-      break;
-    }
-
-    default: tty->print_cr("ERROR: %d: unexpected test_num value.", how);
+    default:
+      // If another number is given, give a generic crash.
+      fatal("Crashing with number %d", how);
   }
   tty->print_cr("VMError::controlled_crash: survived intentional crash. Did you suppress the assert?");
```

Comment:
The message should be changed: `VMError::controlled_crash` -> `controlled_crash`

Reply:
Ok, fixed.